bin/benchmark/Sources/benchmark/Diff/DiffAnalysis.swift

```diff
@@ -0,0 +1,219 @@
+            ("degrees of freedom", tTestResult.degreesOfFreedom.description),
+            ("critical value", footnoteNumberFormatter.string(from: NSNumber(value: tTestResult.criticalValue))!),
+        ]
+        
+        if tTestResult.seriesAreProbablyTheSame {
+            change = .same
+            footnotes = [
+                .init(text: "The before and after values are similar enough that the most probable explanation is that they're random samples from the same data set.", values: footnoteValues)
+            ]
+        } else {
+            let approximateChange = (afterNumbers.mean() - beforeNumbers.mean()) / beforeNumbers.mean()
+            change = .differentNumeric(percentage: approximateChange)
+            
+            footnotes = [
+                .init(text: "The before and after values are different enough that the most probable explanation is that random samples from two different data sets.", values: footnoteValues)
+            ]
+        }
+        
+        if !warnings.isEmpty {
+            footnotes.append(
+                .init(text: "A human should check that the measurements for this metric look like random samples around a certain value to ensure the validity of this result.", values: [])
+            )
+        }
+        
+        return MetricAnalysis(
+            metricName: afterMetric.displayName,
+            metricID: afterMetric.id,
+            change: change,
+            before: beforeMetric.values.formatted(),
+            after: afterMetric.values.formatted(),
+            footnotes: footnotes,
+            warnings: warnings.isEmpty ? nil : warnings
+        )
+    }
+    
+    private static func inputBiasDescription(metric: BenchmarkResultSeries.MetricSeries, sampleName: String, numbers: [Double]) -> String {
+        // Turn the single metric series into an array of single values metric series to render the trend bars.
+        
+        let metricSeries: [BenchmarkResultSeries.MetricSeries]
+        switch metric.values {
+            case .duration(let values):
+                metricSeries = values.map {
+                    .init(id: metric.id, displayName: metric.displayName, values: .duration([$0]))
+                }
+                
+            case .bytesInMemory(let values):
+                metricSeries = values.map {
+                    .init(id: metric.id, displayName: metric.displayName, values: .bytesInMemory([$0]))
+                }
+            case .bytesOnDisk(let values):
+                metricSeries = values.map {
+                    .init(id: metric.id, displayName: metric.displayName, values: .bytesOnDisk([$0]))
+                }
+                
+            case .checksum:
+                // Only numeric metrics support trend bars
+                fatalError("Can't compute input bias for checksum metrics.")
+        }
+        
+        return """
+        The '\(metric.id)' samples from the '\(sampleName)' measurements show _possible_ signs of bias.
```

Comment:
It would be good to clearly state what this entails for the analysis in this sentence.

```suggestion
        warning: Results for the '\(metric.id)' samples for '\(sampleName)' are not normally distributed. Run the benchmark again or inspect the results manually.
```